package.json

```diff
@@ -29,6 +29,7 @@
     "@babel/preset-react": "^7.13.13",
     "@babel/preset-typescript": "^7.15.0",
     "@babel/register": "^7.13.16",
+    "@fullhuman/postcss-purgecss": "^4.0.3",
```

Comment:
Does this spare us the manual selection of modules within bootstrap? Nifty 👏

Reply:
Yes, exactly and it's even better. Instead of hand-picking SCSS files to include (which will also include CSS rules we don't need), we can just create a full, customized build and - based on what selectors are found in the code - it will trim the result. 🧹 

The only downside is, it can't handle dynamic CSS selectors. E.g. if someone does `const className = "my-" + paddingSize`.